config/m4/cuda.m4

```diff
@@ -1 +1,2 @@
 AM_CONDITIONAL([HAVE_CUDA], [true])
+AC_DEFINE([HAVE_CUDA], 1, [cuda enable])
```

Comment:
looks like this PR really depends on the PR which adds configuration flags from cuda, right?

Reply:
yes, will resolve this  once config  flag is in first